build-system/server.js

```diff
@@ -486,6 +487,12 @@ app.use('/a4a(|-3p)/', function(req, res) {
     // This is a special case for testing. `localhost` URLs are transformed to
     // `ads.localhost` to ensure that the iframe is fully x-origin.
     adUrl = urlPrefix.replace('localhost', 'ads.localhost') + adUrl;
+    const inaboxParam = 'inabox=1';
```

Comment:
The issue was that I had to combine it with `/max/` and `/min/` we already use and it was hard to modify the code this way. In a way it's my `/a4a-3p/` that is similar to `/iframe/` and `inabox` is just an additional flag for that.